CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,54 @@
+Shesmu developers is dedicated to providing a harassment-free experience for everyone. We do not tolerate harassment of participants in any form.
```

Comment:
Grammar is a little funky here.

Reply:
They had it set up as a form letter, but assumed it was an institution. I'm going with _The Shesmu Development Team_, but I'm not sure if should be GSI.

Reply:
I think it's fine with 'Shesmu Development Team'. Ideally this would include people from outside OICR/GSI.